_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,287 @@
+---
+id: a25f45
+name: Headers attribute of td element refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute of `td` elements refer to cells in the same `table` element that have a semantic role of `columnheader` or `rowheader`.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any [`headers`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-headers) attribute specified on a [`cell`](https://www.w3.org/TR/html52/tabular-data.html#cell) in a [table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element that is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
This rule probably shouldn't be applicable to `table` elements with a semantic role of `table` or `grid`. If the semantic role of table is changed, the heading assignment no longer applies. I'm not sure if this happens consistently in AT, so that might be worth testing.